src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -106,9 +106,21 @@ protected static void verifyPresentInDatastore(FeedbackQuestionAttributes expect
         if (wildcardId) {
             expected.setId(actual.getId());
         }
+        
+        setEqualTimestampBetweenExpectedAndActual(expected, actual);
+        
```

Comment:
Rather than having methods to modify `createdAt` and `updatedAt`, would it be possible to modify their JSON representation here instead?
This would avoid having methods solely for the sake of testing in production.